Pipfile

```
@@ -7,7 +7,8 @@ name = "pypi"
 apache-airflow = "~=2.5"
 apache-airflow-providers-amazon = "~=7.2"
 sagetasks = "~=0.4"
-py-orca = "~=1.0"
+py-orca = ">1.3.0"
+fs-synapse = "~=1.0"
```

I don't see you using fs-synapse, so I'm guessing you installed it because py-orca complained when you tried using the Synapse hook. Instead of including it here, I suggest that you specify the `all` extra for py-orca as follows. This will install the non-essential dependencies, including fs-synapse. We should update py-orca to inform users to do this if they run into an import error so this feature is more discoverable. 

```suggestion
py-orca = {version = "~=1.3", extras = ["all"]}
```

By the way, if you want to update the `Pipfile.lock` file, you can run `pipenv lock --dev`, which will install the latest version of py-orca and its dependencies. Using `~=1.3` means that it will allow py-orca to progress as long as the major version remains 1. Once version 2.0 is released, pipenv will continue to install version 1.X.